src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1155,40 +1221,47 @@ void ApplicationManagerImpl::StopNaviService(
   app->StopStreaming(service_type);
 }
 
-bool ApplicationManagerImpl::OnServiceStartedCallback(
+void ApplicationManagerImpl::OnServiceStartedCallback(
```

Comment:
Done in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/5fda0d6bf9f2ae414080184699e49c669b73fbf8.